src/object/obj_enum.c

```diff
@@ -468,18 +469,17 @@ enum_pack_cb(daos_handle_t ih, vos_iter_entry_t *entry, vos_iter_type_t type,
 int
 dss_enum_pack(vos_iter_param_t *param, vos_iter_type_t type, bool recursive,
 	      struct vos_iter_anchors *anchors, struct dss_enum_arg *arg,
```

Comment:
We should not have functions prefixed with dss_ in src/object. Please renamed those.